src/main/java/com/softserveinc/dokazovi/config/WebSecurityConfig.java

```diff
@@ -90,38 +96,38 @@ public AuthenticationManager authenticationManagerBean() throws Exception {
 	protected void configure(HttpSecurity http) throws Exception {
 		http
 				.cors()
-					.and()
+				.and()
 				.sessionManagement()
 				.sessionCreationPolicy(STATELESS)
-					.and()
+				.and()
 				.csrf()
-					.disable()
+				.disable()
 				.formLogin().disable()
 				.httpBasic().disable()
 				.logout().disable()
 				.exceptionHandling()
-					.authenticationEntryPoint(new RestAuthenticationEntryPoint())
-					.and()
+				.authenticationEntryPoint(new RestAuthenticationEntryPoint())
+				.and()
 				.authorizeRequests()
 				.antMatchers("/v3/api-docs/**", "/configuration/**", "/swagger*/**", "/webjars/**",
 						"/auth/**", "/oauth2/**")
-					.permitAll()
-				.antMatchers("/user/**", "/post/**", "/tag/**", "/direction/**", "/region/**")
-					.permitAll()
+				.permitAll()
+				.antMatchers(openAPI(USER), openAPI(POST), openAPI(TAG), openAPI(DIRECTION), openAPI(REGION))
+				.permitAll()
 				.anyRequest()
-					.authenticated()
-					.and()
+				.authenticated()
+				.and()
 				.oauth2Login()
-					.authorizationEndpoint()
-						.baseUri("/oauth2/authorize")
-						.authorizationRequestRepository(cookieAuthorizationRequestRepository())
-					.and()
-						.redirectionEndpoint()
-						.baseUri("/login/oauth2/code/*")
-					.and()
-						.userInfoEndpoint()
-						.userService(customOAuth2UserService)
-					.and()
```

Comment:
Could you leave the previous formatting for the current block? It useful for better readability